src/tadox.ts

```diff
@@ -143,4 +165,83 @@ export class TadoX extends Tado {
 
     return this.apiCallX(`/homes/${home_id}/rooms/${room_id}/manualControl`, "post", overlay);
   }
+
+  /**
+   * Set device child lock status.
+   *
+   * @param home_id - The ID of the home for which to fetch the zones.
+   * @param serial_no - The serial number of the device.
+   * @param child_lock - Boolean value to enable or disable the child lock.
+   * @returns A promise that resolves when the operation is complete.
+   */
+  // @ts-expect-error TS2416
```

Comment:
I'm wondering if taking these API calls out of the base class would be better. Then Tado and TadoX implement their own signature for this call

Reply:
That might not help since `TadoX` extends `Tado` at the moment.

Reply:
Oops, I thought I'd fixed it so that `TadoX` did extend `TadoBase`. I flip flopped on that decision a few times and clearly didn't tidy up correctly